airbyte-webapp/src/pages/ConnectionPage/pages/ConnectionItemPage/ConnectionTransformationTab/DbtCloudTransformationsCard.tsx

```diff
@@ -0,0 +1,172 @@
+                          Add transformation
+                        </Button>
+                      </span>
+                    }
+                  >
+                    {hasDbtIntegration ? (
+                      <DbtJobsList jobs={values.jobs} remove={remove} isValid={isValid} />
+                    ) : (
+                      <NoDbtIntegration className={styles.jobListContainer} />
+                    )}
+                  </Card>
+                );
+              }}
+            />
+          </Form>
+        );
+      }}
+    />
+  );
+};
+
+const DbtJobsList = ({
+  jobs,
+  remove,
+  isValid,
+}: {
+  jobs: DbtCloudJob[];
+  remove: (i: number) => void;
+  isValid: boolean;
+}) => (
+  <div className={classNames(styles.jobListContainer, styles.emptyListContent)}>
+    <p className={styles.contextExplanation}>After an Airbyte sync job has completed, the following jobs will run</p>
+    {jobs.length ? (
+      jobs.map((_j, i) => <JobsListItem key={i} jobIndex={i} removeJob={() => remove(i)} />)
+    ) : (
+      <>
+        <img src="/images/octavia/worker.png" alt="An octopus wearing a hard hat, tools at the ready" />
+        No transformations
+      </>
+    )}
+    <div className={styles.jobListButtonGroup}>
+      <Button className={styles.jobListButton} type="reset" variant="secondary">
+        Cancel
+      </Button>
+      <Button className={styles.jobListButton} type="submit" variant="primary" disabled={!isValid}>
+        Save changes
+      </Button>
+    </div>
+  </div>
+);
+
+// TODO give feedback on validation errors (red outline and validation message)
+const JobsListItem = ({ jobIndex, removeJob }: { jobIndex: number; removeJob: () => void }) => {
+  return (
+    <Card className={styles.jobListItem}>
+      <div className={styles.jobListItemIntegrationName}>
+        <img src="/images/external/dbt-bit_tm.png" alt="dbt logo" />
+        dbt Cloud transform
+      </div>
+      <div className={styles.jobListItemInputGroup}>
+        <div className={styles.jobListItemInput}>
+          <Field name={`jobs.${jobIndex}.account`}>
+            {({ field }: FieldProps<string>) => (
+              <>
+                <label htmlFor={`jobs.${jobIndex}.account`} className={styles.jobListItemInputLabel}>
+                  Account ID
+                </label>
+                <Input {...field} type="text" />
+              </>
+            )}
+          </Field>
+        </div>
+        <div className={styles.jobListItemInput}>
+          <Field name={`jobs.${jobIndex}.job`}>
+            {({ field }: FieldProps<string>) => (
+              <>
+                <label htmlFor={`jobs.${jobIndex}.job`} className={styles.jobListItemInputLabel}>
+                  Job ID
+                </label>
+                <Input {...field} type="text" />
+              </>
+            )}
+          </Field>
+        </div>
+        <button type="button" className={styles.jobListItemDelete} onClick={removeJob}>
```

Comment:
Could add an `aria-label` here unless `FontAwesomeIcon` has some text label (I don't think it does?)